cmd/olympus/actions/app.go

```diff
@@ -106,7 +105,8 @@ func App(conf *config.Config) (*buffalo.App, error) {
 	blggr := log.Buffalo(bLogLvl)
 
 	app := buffalo.New(buffalo.Options{
-		Addr: port,
+		Addr: conf.Olympus.Port,
+		Host: "http://127.0.0.1" + conf.Proxy.Port,
```

Comment:
I feel we could remove the colon from the Port config and just use the Host option. What do you think?

Also - I think we should be listening on 0.0.0.0 instead of the loopback by default

Reply:
Ok, that makes sense to change the host to 0.0.0.0. As for the other comment, I'm not sure I completely understand. 
In the config file should I change the `Port = ":3000"` to `Port = "3000"` (for all port related options?) and then change the initialization to:
```
	app := buffalo.New(buffalo.Options{
		Env: ENV,
		PreWares: []buffalo.PreWare{
			cors.Default().Handler,
		},
		SessionName: "_athens_session",
		Logger:      blggr,
		// --- Addr:  line removed ---
		Host: "http://0.0.0.0:" + conf.Proxy.Port,
	})
```
It seems though that doing this though doesn't make it bind to the configured port as the 'Host:' config option doesn't seem to be used when setting up the buffalo app.

Reply:
Yep, let's keep it as port until someone files an issue with a specific reason as to why we need to include the host.